Source/utils/stdcompat/cstddef.hpp

```diff
@@ -1,16 +1,14 @@
 #pragma once
 
 #ifdef __has_include
-#if __has_include(<cstddef>)
 #include <cstddef>
 #if defined(__cplusplus) && __cplusplus >= 201703L
 namespace devilution {
-    using byte = std::byte;
+using byte = std::byte;
 }
 #else
 namespace devilution {
-    using byte = uint8_t;
+using byte = uint8_t;
 }
 #endif
-#endif
 #endif
```

Comment:
This file is missing a trailing newline, which should be enforced by clang-format. Did you run clang-format?

Reply:
I set the IDE to use it Source/.clang-format and there was no rule for that. I'll add the line manually.

Reply:
Ah, actually this rule is part of `.editorconfig`. Which IDE do you use?

Reply:
vscode. It has an option to use .clang-format automatically, but not this .editorconfig, not without addons at least.

Reply:
Please use https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

It is especially important for this project because some files uses CRLF while others use LF (this is configured in `.editorconfig`)

Reply:
Yeah, I just installed that after seeing that file. thx